src/app/components/CallToActionLinkComponent/index.stories.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import CallToActionLink from '.';
+
+export default {
+  title: 'Components/Call To Action Link',
+  parameters: {},
+};
+
+export const ExampleWithChevronAlignedWithText = () => {
```

Comment:
Let's remove the words "Example With" from the names of all stories - when we view the story in Storybook, we just wanted to see "Chevron Aligned with Text" etc.